audit.rdf

```diff
@@ -51,6 +52,19 @@
         updated.</skos:definition>
     <skos:inScheme rdf:resource="http://id.loc.gov/vocabulary/preservation/eventType"/>
     <skos:inScheme rdf:resource="http://id.loc.gov/vocabulary/preservation"/>
+    <skos:broader rdf:resource="http://fedora.info/definitions/v4/audit#resourceModification"/>
+    <rdf:type rdf:resource="http://www.w3.org/2004/02/skos/core#Concept"/>
+  </owl:NamedIndividual>
+
+  <owl:NamedIndividual rdf:about="http://fedora.info/definitions/v4/audit#resourceModification">
+    <rdfs:label xml:lang="en">resource modification</rdfs:label>
+    <skos:prefLabel xml:lang="en">resource modification</skos:prefLabel>
+    <rdfs:comment xml:lang="en">A content file or metadata is modified.</rdfs:comment>
+    <skos:definition xml:lang="en">A content file or metadata is modified.</skos:definition>
+    <skos:inScheme rdf:resource="http://id.loc.gov/vocabulary/preservation/eventType"/>
+    <skos:inScheme rdf:resource="http://id.loc.gov/vocabulary/preservation"/>
+    <skos:narrower rdf:resource="http://fedora.info/definitions/v4/audit#bitstreamModification"/>
```

Comment:
Should this be:
`<skos:narrower rdf:resource="http://fedora.info/definitions/v4/audit#contentModification"/>`
??